packages/contractkit/src/wrappers/Validators.ts

```diff
@@ -343,6 +354,35 @@ export class ValidatorsWrapper extends BaseWrapper<Validators> {
     this.contract.methods.affiliate
   )
 
+  async getStatus(
+    validator: Address,
+    blocks: any[],
```

Comment:
I believe the type here can be `Block[]` with `import { Block } from 'web3/eth/types'`
https://github.com/ethereum/web3.js/blob/c0e727eac7c16826c39fd43a03a4930b5ff6fff8/packages/web3/types.d.ts#L418